pkg/agent/agentclient/clientset.go

```diff
@@ -128,30 +128,48 @@ func (cs *ClientSet) newAgentClient() (*AgentClient, error) {
 	return newAgentClient(cs.address, cs.agentID, cs, cs.dialOption)
 }
 
+func (cs *ClientSet) resetBackoff() *wait.Backoff {
+	return &wait.Backoff{
+		Steps:    3,
+		Jitter:   0.1,
+		Factor:   1.5,
+		Duration: cs.syncInterval,
+	}
```

Comment:
Do you want initialize "Cap" too, just in  case it grows too much?

Reply:
+1

Reply:
Done